velox/exec/Exchange.cpp

```diff
@@ -14,13 +14,16 @@
  * limitations under the License.
  */
 #include "velox/exec/Exchange.h"
+#include <velox/buffer/Buffer.h>
+#include <velox/common/base/Exceptions.h>
+#include <velox/common/memory/MappedMemory.h>
 #include "velox/exec/PartitionedOutputBufferManager.h"
 
 namespace facebook::velox::exec {
 
 SerializedPage::SerializedPage(std::unique_ptr<folly::IOBuf> iobuf)
```

Comment:
The existing SerializedPage ctor can simply call new SerializedPage ctor with null memory pool to reduce the code copy? Do we want to enforce to pass memory pool to construct SerializedPage from iobuf next? Thanks!